RELEASING.md

```diff
@@ -172,33 +182,91 @@ and improve it. We want small, constant improvements which compound. Therefore:
 > SDK. This will ensure that all the APIs in the SDK are also available in the
 > Engine it depends on.
 
-- [ ] Create e.g. `.changes/v0.11.8.md` by either running `changie batch patch`
-      (or `changie batch minor` if this is a new minor).
-
 > [!NOTE]
 >
 > If you do not have `changie` installed, see https://changie.dev
 
-- [ ] Make any necessary edits to the newly generated file, e.g.
-      `.changes/v0.11.8.md`
-- [ ] Update `CHANGELOG.md` by running `changie merge`.
-- [ ] `30 mins` Submit a PR - e.g. `add-v0.11.8-release-notes` with the new release notes
-      so that they can be used in the new release. Get the PR reviewed & merged.
-      The merge commit is what gets tagged in the next step.
-- [ ] Ensure that all checks are green ✅ for the `<ENGINE_GIT_SHA>` on the
-      `main` branch that you are about to release.
-- [ ] `30mins` When you have confirmed that all checks are green, run the following:
+Setup some variables used throughout the release process:
 
 ```console
-git checkout main
+# 🚨 change this from `main` to `release-vX.Y.Z` if releasing off a non-main branch
+export RELEASE_BRANCH=main
 
 # If not named "origin" in your local checkout, replace "origin" with whatever the
 # github.com/dagger/dagger repo is named for you locally
 export DAGGER_REPO_REMOTE=origin
+```
+
+> [!NOTE]
+>
+> This process involves direct pushes to the github.com/dagger/dagger repository. Recently there have been permission issues here that result in errors like
+>
+> `refusing to allow an OAuth App to create or update workflow` when pushing.
+>
+> If you encounter this, you may need to change your git remote from https to ssh, e.g.
+>
+> `git remote set-url $DAGGER_REPO_REMOTE git@github.com:dagger/dagger.git`
+>
+> It's advisable to go back to https after the release is complete to not retain excessive permissions longer than necessary, e.g. to go back run
+>
+> `git remote set-url $DAGGER_REPO_REMOTE https://github.com/dagger/dagger.git`
```

Comment:
Hm, this one is weird. Not sure why this happens, I generally use `ssh` remotes everywhere, I've never seen this one.

cc @gerhard have we changed permissions at all recently?

Reply:
Oddly it was inconsistent when I got it. I pushed the `sdk/go/v0.11.9` tag no problem but then hit this when pushing python in the next steps?